src/pages/docs/guides/chat/build-embedded-chat.mdx

```diff
@@ -0,0 +1,414 @@
+* **Context preservation:** Users returning to a task don't lose important information.
+* **Audit trail:** Complete conversation records for compliance, training, or dispute resolution.
+
+History is almost always beneficial:
+
+* **Support tickets:** Agents can see the full conversation history to understand the issue, after handover from automated/AI support or another agent.
+* **Collaborative work:** Team members can catch up on decisions and progress.
+* **Gaming sessions:** Players can review moves and strategy discussions.
+* **Marketplace transactions:** Complete communication record for orders and deliveries.
+
+<Code>
+```javascript
+// Get the chat room
+const room = await chatClient.rooms.get('support-ticket-123');
+
+// Subscribe to messages
+const subscription = room.messages.subscribe((messageEvent) => {
+  console.log('Received:', messageEvent);
+  // handle message event to update state and UI
+});
+
+// attach the room
+await room.attach();
+
+// Load recent history for context
+// This ensures you get a complete picture without missing messages
+let historyResponse = await subscription.historyBeforeSubscription({limit: 50});
+
+// Iterate through all pages
+while (true) {
+  console.log('Messages: ', historyResponse.items);
+  if (historyResponse.hasNext()) {
+    historyResponse = await historyResponse.next();
+  } else {
+    break;
+  }
+}
+```
+</Code>
+
+## React UI Kit
+
+The [React UI Kit](/docs/chat/react-ui-kit) provides ready-made components for building chat interfaces. Drop in the [App](/docs/chat/react-ui-kit/components#app) component to see a working chat UI within minutes, or use individual components like [ChatWindow](/docs/chat/react-ui-kit/components#chat-window), [ChatMessageList](/docs/chat/react-ui-kit/components#chat-message-list), [MessageInput](/docs/chat/react-ui-kit/components#message-input), and [ParticipantList](/docs/chat/react-ui-kit/components#participant-list) to build custom interfaces.
+
+Components handle message display with history loading, editing and deletion, reactions, typing indicators, and presence. [Providers](/docs/chat/react-ui-kit/providers) manage themes, avatars, and chat settings. See the [getting started guide](/docs/chat/getting-started/react-ui-kit) for setup details.
+
+## Enriching experiences with Ably's realtime services
+
+Ably's comprehensive platform enables you to combine chat with other realtime features to create rich, interactive experiences.
+
+[**Pub/Sub channels**](/docs/basics) enable you to build rich, interactive experiences:
+- **Live polls:** Quick feedback during collaborative decisions.
+- **Status updates:** Real-time progress indicators for tasks.
+- **Interactive ratings:** Instant feedback collection.
+- **Live auctions:** Real-time bidding in marketplace scenarios.
+
+These combined services transform basic chat into comprehensive task management platforms, where communication, coordination, and real-time updates work together seamlessly.
```

Comment:
I'm really wary of including this. We're basically introducing the entire Pub/Sub product docs for someone to go and read with no context around channels vs rooms and that they can't be shared. 

I think this is a product of a combined use case guide again if I'm being honest. If this was a 1:1 guide for example then this section is irrelevant. To some extent even small group chats - these are more suited to when you're embedding chat in an app, but would someone be coming here for 'embedded chat' if they didn't already have an app?

Reply:
I think it's a unique selling point of Ably vs other providers that we can do this. Happy to discuss if you think we should get rid of this section? [Previous feedback](https://github.com/ably/docs/pull/2892#discussion_r2459764362) on it was to move it further up.